src/components/ComponentDoc/ComponentDoc.sass

```diff
@@ -1 +1,64 @@
-@import '../../styles/core'
\ No newline at end of file
+@import '../../styles/core'
+
+section
+    hr
+        border-bottom: 1px solid black
+    
+    h2
+        font-size: 2.5rem
+        line-height: 4rem
+        font-weight: 400
+
+    h3
+        font-size: 2rem
+        line-height: 4rem
+        font-weight: 400
+
+    table
+        font-size: 1.5rem
+        font-weight: 400
+        display: flex
```

Comment:
lol what are you trying to do to this table? Tables by default are `display: table`, maybe this would be better with divs instead?